web-animations/testcommon.js

```diff
@@ -56,3 +61,12 @@ function removeElement(element) {
 function type(object) {
     return Object.prototype.toString.call(object).slice(8, -1);
 }
+
+// Returns expected top of the target element at currentTime
+function getExpectedTop(currentTime, endTime) {
+    if (!endTime) {
+        endTime = ANIMATION_DURATION;
+    }
+    return ANIMATION_TOP_0 + (ANIMATION_TOP_1 - ANIMATION_TOP_0)
+        * (currentTime % endTime);
```

Comment:
I think currentTime % endTime is Ok here. Example. ANIMATION_TOP_0 = 0, ANIMATION_TOP_1 = 100, endTime = 1000, currentTime = 1100. At 1100 expected value is 10, not 110. That's how tests assumes.  For the case when iterationComposite = 'accumulate' we use code like 
            assert_approx_equals(window.getComputedStyle(target).top,
                N * ANIMATION_TOP_1 + getExpectedTop(...), COMPUTED_STYLE_EPSILON,  '...');
;